virtcontainers/persist/api/device.go

```diff
@@ -66,6 +66,10 @@ type VhostUserDeviceAttrs struct {
 
 	// MacAddress is only meaningful for vhost user net device
 	MacAddress string
+
+	// PCIAddr is the PCI address used to identify the slot at which the drive is attached.
+	// It is only meaningful for vhost user devices
+	PCIAddr string
```

Comment:
s/only meaningful for vhost user devices/ only meaningful for vhost user block devices/

Reply:
Ack

Reply:
Sorry, missed it.